core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml

```diff
@@ -845,3 +845,65 @@ specification:
        destination_port_range: "0"
        source_address_prefix: "0.0.0.0/0"
        destination_address_prefix: "0.0.0.0/0"       
+---
+kind: infrastructure/virtual-machine
+title: "Virtual Machine Infra"
+provider: aws
+name: repository-machine
+specification:
+  size: t2.medium
+  authorized_to_efs: true
+  mount_efs: true
+  security:
+    rules:
```

Comment:
For the security rules pls remove:

- priority
- access
- source_port_range

As they are not needed for AWS

Reply:
To clarify check this pull request: #1714

Reply:
ok, done